config/config.yaml

```diff
@@ -8,9 +8,13 @@ dirs:
   processed: processed
   final: final
   models: models
+  phoneme_dir: config/phonemes.json
```

Comment:
This doesn't work on windows. If we want to use it as a configuration file then we store `phonemes` as a yaml - Hydra supports nested config files.